store/store.go

```diff
@@ -1391,3 +1409,12 @@ func (s *Store) PaymentMethods(user *auth.UserState) (*PaymentInformation, error
 		return nil, fmt.Errorf("cannot get payment methods: unexpected HTTP code %d%s", resp.StatusCode, details)
 	}
 }
+
+func (s *Store) listRefreshFields() []string {
+	fields := append([]string{}, s.detailFields...)
```

Comment:
interestingly this avoids a copy even when you need to append, because fields' capacity will already be at least one bigger than its length.